### ISSUE TITLE
Redrawing of views

### ISSUE THREAD

**Issue by iankoex** [exyte/PopupView#55]
Changing how the view is shown would prevent reloading of the content.
```
public func body(content: Content) -> some View {
        Group {
            main(content: content)
        }
        .valueChanged(value: isPresented) { isPresented in
            appearAction(isPresented: isPresented)
        }
    }
    
    private func main(content: Content) -> some View {
        ZStack {
            content
                .frameGetter($presenterContentRect)
            
            if showContent {
                backgroundColor
                    .applyIf(closeOnTapOutside) { view in
                        view.contentShape(Rectangle())
                    }
                    .addTapIfNotTV(if: closeOnTapOutside) {
                        dismiss()
                    }
                    .edgesIgnoringSafeArea(.all)
                    .opacity(currentBackgroundOpacity)
                    .animation(animation)
            }
        }
        .overlay(sheet())
    }
```

**Comment by f3dm76** [comment]
Hello @iankoex, sorry, I'm confused - is this piece of code a proposed change to the lib? I mean, do you want me to include this into project to prevent reloading of the content? Why would you want to prevent content reloading? Would you please specify what problem exactly you are having, what is the desired behaviour, and what is the proposed solution, as is customary.

**Comment by iankoex** [comment]
If the popup is triggered in a scrollview, the content in this case the scrollview reloads. This makes the scrollview go to the top, offset of zero. The snippet above does not reload the content. The desired behaviour is that when the popup is shown, the scrollview maintains its position(offset).

**Comment by f3dm76** [comment]
Hey again @iankoex, got your point, however your suggested solution kind of beats the purpose of lazy load - .overlay(sheet()) is called in any case in your scenario. Keeping in mind your described problem, I restructured a little. I included my solution into 1.0.4 release, please give it a go, and do let me know if that didn't fix your problem. Have a great day!

**Comment by f3dm76** [comment]
Hello @iankoex, it's been a while so I'm gonna this issue, but please do let me know if the problem is not fixed. Have a good day!
